docs/staging.asciidoc

```diff
@@ -165,6 +165,11 @@ osc staging select --move B 12345
 This command will move #12345 from openSUSE:Factory:Staging:A to
 openSUSE:Factory:Staging:B
 
+You can also merge staging projects by giving other staging projects as arguments:
+
+--------------------------------------------------------------------------------
+osc staging select --move B A C
```

Comment:
I'd either write this as `osc staging select B --move A C` or add a further comment that this command merges A and C into B

Reply:
As you wish, not that I think anyone ever read this file :)

Reply:
like every doc; docs are just written so we can say RTFM